### ISSUE TITLE
Using all template filters from jinja

### ISSUE THREAD

**Issue by MikeVL** [niwibe/django-jinja#101]
In latest version of django-jinja removed JINJA2_FILTERS_REPLACE_FROM_DJANGO.

How i can use `value|default('', true)` from jinja ljib?

**Comment by niwibe** [comment]
Hi @MikeVL 

In the new version, all the things are managed using jinja extensions. The extensions that manages the replacement of jinja filters with django versions is "django_jinja.builtins.extensions.DjangoExtraFiltersExtension". You can verwrite the list with with a concrete list of extensions that you need.

Related documentation: http://niwibe.github.io/django-jinja/#_add_additional_extensions

**Comment by MikeVL** [comment]
Now my settings is:
```
JINJA2_EXTENSIONS = (
    'cacheops.jinja2.cache',
    'pipeline.jinja2.ext.PipelineExtension',
)
```
And i can`t override DEFAULT_EXTENSIONS and exclude `django_jinja.builtins.extensions.DjangoExtraFiltersExtension`
```
DEFAULT_EXTENSIONS = [
    "jinja2.ext.do",
    "jinja2.ext.loopcontrols",
    "jinja2.ext.with_",
    "jinja2.ext.i18n",
    "jinja2.ext.autoescape",
    "django_jinja.builtins.extensions.CsrfExtension",
    "django_jinja.builtins.extensions.CacheExtension",
    "django_jinja.builtins.extensions.TimezoneExtension",
    "django_jinja.builtins.extensions.UrlsExtension",
    "django_jinja.builtins.extensions.StaticFilesExtension",
    "django_jinja.builtins.extensions.DjangoFiltersExtension",
    "django_jinja.builtins.extensions.DjangoExtraFiltersExtension",
]
   ...

    initial_params["extensions"].extend(DEFAULT_EXTENSIONS)
    initial_params["extensions"].extend(JINJA2_EXTENSIONS)
```

**Comment by niwibe** [comment]
I not clearly understand you.

The simplest way to define the concrete list of extensions you need is:

```python
JINJA2_EXTENSIONS = [
    "jinja2.ext.do",
    "jinja2.ext.loopcontrols",
    "jinja2.ext.with_",
    "jinja2.ext.i18n",
    "jinja2.ext.autoescape",
    "django_jinja.builtins.extensions.CsrfExtension",
    "django_jinja.builtins.extensions.CacheExtension",
    "django_jinja.builtins.extensions.TimezoneExtension",
    "django_jinja.builtins.extensions.UrlsExtension",
    "django_jinja.builtins.extensions.StaticFilesExtension",
    "django_jinja.builtins.extensions.DjangoFiltersExtension",
]
```

Furthermore, I'm thinking remove the django extra filters extension from default extensions list. I will release the new version with the change today. Using the next (1.3.1) you should do nothing for get the desired behavior.

**Comment by MikeVL** [comment]
I setup this settings:
```

JINJA2_EXTENSIONS = (
    'jinja2.ext.do',
    'jinja2.ext.loopcontrols',
    'jinja2.ext.with_',
    'jinja2.ext.i18n',
    'jinja2.ext.autoescape',
    'django_jinja.builtins.extensions.CsrfExtension',
    'django_jinja.builtins.extensions.CacheExtension',
    'django_jinja.builtins.extensions.TimezoneExtension',
    'django_jinja.builtins.extensions.UrlsExtension',
    'django_jinja.builtins.extensions.StaticFilesExtension',
    'django_jinja.builtins.extensions.DjangoFiltersExtension',
    'cacheops.jinja2.cache',
    'pipeline.jinja2.ext.PipelineExtension',
)
```
Results:
```
    initial_params.setdefault("extensions", [])
    initial_params["extensions"].extend(DEFAULT_EXTENSIONS)
    initial_params["extensions"].extend(JINJA2_EXTENSIONS)
    print initial_params["extensions"]
```
```
[
    'jinja2.ext.do',
    'jinja2.ext.loopcontrols',
    'jinja2.ext.with_',
    'jinja2.ext.i18n',
    'jinja2.ext.autoescape',
    'django_jinja.builtins.extensions.CsrfExtension',
    'django_jinja.builtins.extensions.CacheExtension',
    'django_jinja.builtins.extensions.TimezoneExtension',
    'django_jinja.builtins.extensions.UrlsExtension',
    'django_jinja.builtins.extensions.StaticFilesExtension',
    'django_jinja.builtins.extensions.DjangoFiltersExtension',
    'django_jinja.builtins.extensions.DjangoExtraFiltersExtension',
    'jinja2.ext.do',
    'jinja2.ext.loopcontrols',
    'jinja2.ext.with_',
    'jinja2.ext.i18n',
    'jinja2.ext.autoescape',
    'django_jinja.builtins.extensions.CsrfExtension',
    'django_jinja.builtins.extensions.CacheExtension',
    'django_jinja.builtins.extensions.TimezoneExtension',
    'django_jinja.builtins.extensions.UrlsExtension',
    'django_jinja.builtins.extensions.StaticFilesExtension',
    'django_jinja.builtins.extensions.DjangoFiltersExtension',
    'cacheops.jinja2.cache',
    'pipeline.jinja2.ext.PipelineExtension'
]
```

**Comment by MikeVL** [comment]
May be change code to this:
```
initial_params["extensions"].extend(getattr(settings, 'JINJA2_EXTENSIONS', DEFAULT_EXTENSIONS)
```

**Comment by niwibe** [comment]
Hmm, yes you are right, seems a bug. I will fix it this afternoon.

**Comment by niwibe** [comment]
This already fixed in 1.3.1 and extra filters removed from default.
